policyengine_us/tests/policy/baseline/gov/usda/snap/snap_min_allotment.yaml

```diff
@@ -17,14 +17,23 @@
     # Ineligible as a family of 3.
     snap_min_allotment: 0
 
+- name: NJ provides a different minimum allotment amount
+  period: 2024
+  input:
+    spm_unit_size: 2
+    state_code: NJ
+    state_group: CONTIGUOUS_US
+  output:
+    snap_min_allotment: 95 * 12  
+
 - name: DC provides a different minimum allotment amount
-  period: 2024-01
+  period: 2024
```

Comment:
Can we keep the period as a month? For monthly variables it better aligns with our minimalist approach. Please change others in this file too.

Reply:
>Please change others in this file too.